pipelines/optimus/README.md

```diff
@@ -1,33 +1,223 @@
+| Aligner           |STAR       |[Dobin, et al.,2013](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3530905/)|
+| Transcript Quantification |Utilities for processing large-scale single cell datasets |[Sctools](https://github.com/HumanCellAtlas/sctools) |                      
+|Data Input File Format |File format in which sequencing data is provided |[FASTQ](https://academic.oup.com/nar/article/38/6/1767/3112533) |                       
+|Data Output File Format |File formats in which Optimus output is provided |[BAM](http://samtools.github.io/hts-specs/), [Zarr version 2](https://zarr.readthedocs.io/en/stable/spec/v2.html), Python numpy arrays (internal), [Loom](http://loompy.org/) |
+
+# Set-up
+## Optimus Installation and Requirements
+The Optimus pipeline code can be downloaded by cloning the github repository [Skylab](https://github.com/HumanCellAtlas/skylab/). For the latest release of Optimus, please see the realease tags prefixed with "optimus" [here](https://github.com/HumanCellAtlas/skylab/releases). 
 
-### Requirements  
 Optimus can be deployed using [Cromwell](https://software.broadinstitute.org/wdl/), a GA4GH compliant, flexible workflow management system that supports multiple computing platforms. Optimus can also be run in [Terra](https://app.terra.bio/#workspaces/help-gatk/HCA_Optimus_Pipeline), a cloud-based analysis platform. In this featured workspace the user will find the Optimus pipeline, configurations, required reference data and other inputs, and example testing data.
 
-### Running Optimus
+## Inputs
+
+The necessary inputs for the Optimus pipeline are detailed in a json file, such as in this [example](https://github.com/HumanCellAtlas/skylab/blob/master/pipelines/optimus/example_test_inputs.json). 
+
+### Sample Data Input
+
+Each 10X v2 and v3 3’ sequencing experiment generates triplets of Fastq files for any given sample:  
+
+1. A forward reads (r1_fastq), containing the unique molecular identifier (UMI) and cell barcode sequences
+2. A reverse reads (r2_fastq), which contain the alignable genomic information from the mRNA transcript 
+3. An index fastq (i1_fastq) that contains the sample barcodes, when provided by the sequencing facility
+
+Note: Optimus is currently a single sample pipeline, but can take in multiple sets of fastqs for a sample that has been split over lanes of sequencing. 
+
+### Additional Reference Inputs
+
+The json file also contains metadata for the folowing reference information:
+
+* Whitelist: a list of known cell barcodes from [10X genomics](https://www.10xgenomics.com/)
+* Star reference genome for human or mouse built with a wdl [here]( https://github.com/HumanCellAtlas/skylab/blob/master/library/accessory_workflows/build_star_reference/BuildStarReferenceBundle.wdl)
+* Sample_id: Name of sample matching file
```

Comment:
Which one did you find confusing? I ask because I initially had a problem with sample ID, but I was thinking you might not like the STAR reference genome?